.travis.yml

```diff
@@ -0,0 +1,42 @@
+jobs:
+  include:
+    - language: node_js # js testing
+
+      node_js:
+      - "stable" # latest stable node version
```

Comment:
Pinpoint to exact version as being used in this project or else there will be engine clash!

Reply:
@parthsujalshah work on this